packages/dds/tree/src/feature-libraries/chunked-forest/emptyChunk.ts

```diff
@@ -0,0 +1,92 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { FieldKey, CursorLocationType, FieldUpPath, UpPath } from "../../core";
+import { fail } from "../../util";
+import { ChunkedCursor, cursorChunk, dummyRoot, TreeChunk } from "./chunk";
+
+export const emptyChunk: TreeChunk = {
+    topLevelLength: 0,
+    cursor(): ChunkedCursor {
+        return emptyCursor;
+    },
+    referenceAdded(): void {},
+    referenceRemoved(): void {},
+    isShared(): boolean {
+        return false; // Immutable, so sharing does not matter.
+    },
+};
+
+const emptyPath: FieldUpPath = {
+    parent: undefined,
+    field: dummyRoot,
+};
+
+/**
+ * Cursor over an empty field.
+ * Contains no nodes and is stateless.
```

Comment:
I added docs to emptyChunk which cover this somewhat. I suspect the main use will be the return value of chunk tree when given an empty tree (once we make it take in a field instead of a node for the root).